packages/mobile/src/components/fields/PasswordField.tsx

```diff
@@ -22,7 +22,6 @@ export const PasswordField = (props: PasswordFieldProps) => {
     <PasswordInput
       {...field}
       error={hasError}
-      helperText={hasError ? error : undefined}
```

Comment:
Matches pattern on web where we dont show any text since the completion checks handle that